src/string/escape/index.js

```diff
@@ -0,0 +1,8 @@
+function escape (string) {
+  if (typeof (string) !== 'string') {
```

Comment:
1. Use a single(last) return only
2. Let's Improve how it works. Here is an example.
https://jsfiddle.net/vitaliymaz/Lua14fop/4/

Reply:
https://jsfiddle.net/vitaliymaz/Lua14fop/5/